applications/main/infrared/infrared_cli.c

```diff
@@ -328,6 +344,167 @@ static void infrared_cli_process_decode(Cli* cli, FuriString* args) {
     furi_record_close(RECORD_STORAGE);
 }
 
+static void infrared_cli_process_remote(Cli* cli, FuriString* args) {
+    enum RemoteTypes Remote;
+
+    FuriString* command;
+    FuriString* remote;
+    FuriString* signal;
+    command = furi_string_alloc();
+    remote = furi_string_alloc();
+    signal = furi_string_alloc();
+
+    do {
+        if(!args_read_string_and_trim(args, command) || args_length(args) > 5) {
+            infrared_cli_print_usage();
+            break;
+        }
+
+        if(strncmp(furi_string_get_cstr(command), "list", 4) == 0) {
+            args_read_string_and_trim(args, remote);
+            if(strncmp(furi_string_get_cstr(remote), "tv", 2) == 0) {
+                Remote = TV;
+            } else if(strncmp(furi_string_get_cstr(remote), "ac", 2) == 0) {
+                Remote = AC;
+            } else {
+                printf("Invalid remote type.\r\n");
+                break;
+            }
+            infrared_cli_list_remote_signals(Remote);
+            break;
+        }
+
+        if(strncmp(furi_string_get_cstr(command), "tv", 2) == 0) {
```

Comment:
When comparing a `FuriString`, use `furi_string_cmp()` or `furi_string_cmp_str()` (depending on the second operand).

Reply:
Changed compare functionn to use` furi_string_cmp_str()`